templates/ts-empty/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "ts-project-typescript",
+    "version": "0.0.1",
+    "type": "module",
+    "description": "This is an example of an Apify actor.",
+    "engines": {
+        "node": ">=16.0.0"
+    },
+    "dependencies": {
+        "apify": "^3.1.4"
```

Comment:
Maybe we could add Crawlee here. That's something that Lukas mentioned. That sometimes you want to immediately work with it and you have to rebuild.